data/moves.js

```diff
@@ -1103,6 +1104,7 @@ exports.BattleMovedex = {
 		desc: "If the user is hit by a contact move this turn before it can execute this move, the attacker is burned.",
 		shortDesc: "Burns on contact with the user before it moves.",
 		id: "beakblast",
+		isViable: true,
```

Comment:
Explanation: Toucannon is quite slow, and this is a reasonable choice for it if player doesn't want recoil from Brave Bird.